apps/condo/package.json

```diff
@@ -17,7 +17,13 @@
     "createservice": "node ./../../bin/createservice",
     "createschema": "node ./../../bin/createschema",
     "lint-schema": "NODE_ENV=test node ./../../bin/lint-schema.js",
-    "lint-i18n-translations": "node ./../../bin/lint-i18n-translations.js"
+    "lint-i18n-translations": "node ./../../bin/lint-i18n-translations.js",
+    "refresh-sbbol-tokens": "node ./../../bin/sbbol.js",
+    "create-integration": "node ./../../bin/create-integration.js",
+    "sbbol:crypto:get": "node ./../../bin/sbbol/crypto.js",
+    "fix:ticket:contacts": "node ./../../bin/fix-tickets-contacts.js",
+    "fix:meterReadings:contacts": "node ./../../bin/fix-meter-readings-contacts.js",
+    "subscription:prolong": "node ../../bin/prolong-subscription.js"
```

Comment:
It was removed in PR #992
Rebase, please.

Reply:
done